src/renderer/api/catalog-entity-registry.ts

```diff
@@ -39,14 +41,6 @@ export class CatalogEntityRegistry {
     });
   }
 
-  set activeEntity(entity: CatalogEntity) {
```

Comment:
This is removed cause doesn't add any extra logic on `set`. 
We can revert when we need more actions on assign.